chunky/src/java/se/llbit/chunky/world/EmptyChunk.java

```diff
@@ -85,7 +78,7 @@ private void renderEmpty(MapTile tile) {
     // Do nothing.
   }
 
-  @Override public synchronized boolean loadChunk(ChunkData chunkData, int yMin, int yMax) {
+  @Override public synchronized boolean loadChunk(Mutable<ChunkData> chunkData, int yMin, int yMax) {
```

Comment:
Does this _have_ to be mutable? Could we somehow refactor this to return a pair of ChunkData and boolean? Or ChunkData/null?